addons/bus/static/src/workers/websocket_worker.js

```diff
@@ -332,7 +332,7 @@ export class WebsocketWorker {
      * applied to the reconnect attempts.
      */
     _retryConnectionWithDelay() {
-        this.connectRetryDelay = this.connectRetryDelay * 1.5 + 500 * Math.random();
+        this.connectRetryDelay = this.connectRetryDelay * 1.5 + 1000 * Math.random();
```

Comment:
so 1.5 is the exponential back-off and 1000*Math.random() is the jitter right ?

Reply:
Yes :)